### PR TITLE
Add FAQ entry w.r.t. JavaScript lexing

### PR DESCRIPTION
**Pull request by mathiasbynens**
@erights PTAL

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -235,6 +235,10 @@ We considered also proposing an operator for symmetric difference (see [issue #5
 
 Instead, we are proposing to reserve doubled ASCII punctuation and symbols for future use. That will allow for future proposals to add `~~` for example, as suggested in [UTS \#18](https://www.unicode.org/reports/tr18/#Subtraction_and_Intersection), for symmetric difference.
 
+### Does this proposal affect JavaScript lexing?
+
+It’s an explicit goal of our proposal that a correct JavaScript lexer before this proposal remains a correct JavaScript lexer after this proposal.
```
Comment (mathiasbynens) [inline README.md:240]:
```suggestion
### Does this proposal affect ECMAScript lexing?

It’s an explicit goal of our proposal that a correct ECMAScript lexer before this proposal remains a correct ECMAScript lexer after this proposal.
```

File: README.md [hunk 2]
```diff
@@ -235,6 +235,10 @@ We considered also proposing an operator for symmetric difference (see [issue #5
 
 Instead, we are proposing to reserve doubled ASCII punctuation and symbols for future use. That will allow for future proposals to add `~~` for example, as suggested in [UTS \#18](https://www.unicode.org/reports/tr18/#Subtraction_and_Intersection), for symmetric difference.
 
+### Does this proposal affect ECMAScript lexing?
+
+It’s an explicit goal of our proposal that a correct ECMAScript lexer before this proposal remains a correct ECMAScript lexer after this proposal.
```
Comment (mathiasbynens) [inline README.md:240]:
```suggestion
No. It’s an explicit goal of our proposal that a correct ECMAScript lexer before this proposal remains a correct ECMAScript lexer after this proposal.
```

**Review by markusicu — APPROVED** [review]
lgtm

**Review by erights — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**